src/Protocol/Protocol_1_9.cpp

```diff
@@ -973,11 +978,14 @@ void cProtocol_1_9_0::SendPlayerListAddPlayer(const cPlayer & a_Player)
 	Pkt.WriteVarInt32(1);
 	Pkt.WriteUUID(a_Player.GetUUID());
 	Pkt.WriteString(a_Player.GetPlayerListName());
+	LOGD("Sending player tab list for \"%s\"...", a_Player.GetPlayerListName());
```

Comment:
This functions is called once for every player in the server, that's why I think these `LOGD`s would be too spammy.